### ISSUE TITLE
LOOT suddenly informs me that I miss the patches -  I have installed

### ISSUE THREAD

**Issue by Giboss** [loot/loot#1117]
[LOOTDebugLog.txt](https://github.com/loot/loot/files/3289422/LOOTDebugLog.txt)

Hello...

Weird...

I have installed:
Economy Overhaul and Speechcraft Improvements
Cutting Room Floor
Coking in Skyrim

I have installed 2 patch Economy + CRF and Economy + CiS.

Until now everything was fine, suddenly LOOT shows me:

EconomyOverhaulandSpeechcraftImprovements.esp
-You seem to be using Cooking In Skyrim, but you have not enabled a compatibility patch for this mod. A compatibility patch is included with this plugin's installer.
-You seem to be using Cutting Room Floor, but you have not enabled a compatibility patch for this mod. A compatibility patch is included with this plugin's installer.

Both patches are enable, Skyrim does not show that anything is missing.

Regards

**Comment by CptMcSplody** [comment]
@Giboss Those messages are only checking for the latest plugin name.

```
[09:02:25.079243] [trace]: Checking if plugin "EconomyOverhaulandCuttingRoomFloorPatch.esp" is active.
[09:02:25.079268] [trace]: Checking if plugin "mihailwolvesofcyrodiil.esp" is active.
[09:02:25.079296] [trace]: Checking if plugin "Qw_S3DTrees_CRF Patch.esp" is active.
[09:02:25.079554] [trace]: Checking if plugin "RSC_USSEP_AventusFix.esp" is active.
[09:02:25.079587] [trace]: Checking if plugin "VampireAttacksRestored.esp" is active.
[09:02:25.079617] [trace]: Checking if plugin "EconomyOverhaul CV - Missives Patch.esp" is active.
[09:02:25.079642] [trace]: Checking if plugin "cookinginskyrimeconomyoverhaulpatch.esp" is active.
[09:02:25.079667] [trace]: Checking if plugin "EconomyOverhaul - GeneralMerchantsBuyEverything.esp" is active.
```

It appears that older versions of these patches had a different name,
`EconomyOverhaulandCuttingRoomFloorPatch.esp`,
`cookinginskyrimeconomyoverhaulpatch.esp`.

Latest version of Economy Overhaul has these plugins
`EconomyOverhaul CV - CuttingRoomFloor Patch.esp`
`EconomyOverhaul LV - CuttingRoomFloor Patch.esp`
EconomyOverhaul CV - CookingInSkyrim Patch.esp


You can ignore the message if you're just renaming the patches.

**Comment by Giboss** [comment]
That's right, the author has changed the names of patches. I remembered.

Huge thanks for help :)

**Comment by UnusualParticle** [comment]
I'm glad that you figured out your problem :) I just wanted to add a comment here for anyone who has the same problem I did.

I installed the mods and the patches, and LOOT still gave me the error. Found out that though they were installed and enabled in MO2 (the main mod list on the left), the .esp's (or .esl's, I don't remember) were not activated in the plugin list (one of the tabs on the right). As soon as I activated them, the LOOT messages went away. Happy modding!
